projects/catton/index.js

```diff
@@ -0,0 +1,42 @@
+const { call, getJettonBalances, getTonBalance, sumTokens } = require("../helper/chain/ton");
+const { sleep } = require('../helper/utils')
+const ADDRESSES = require('../helper/coreAssets.json')
+
+const TON_POOL_ADDRESS = "EQAze1aSZHY1yUGz1BFndH62k-VYpXYeDiYofCXTRZClF8Qr"
+const USD_POOL_ADDRESS = "EQBy7pjr6IBzqW8vuVCZ780evtnkiIF3jZSRRDxeqScfZoU9"
+const CTON_ADDRESS = "0:86cf8401d283627a87b58c367b440cad933ab8aa7b383419e8ff7d1a00c945f8"
+const CTUSDT_ADDRESS = "0:9fb449ce8fb43d0f682c713c01d9d8357c7cd0d4a49dd64dd585926990174a4e"
+const USDT_ADDRESS = "0:b113a994b5024a16719f69139328eb759596c38a25f59028b146fecdc3621dfe"
+
+async function tvl(api) {
+
+ /*  const vault = await call({ target: "UQDFlyZ5zsWyowbZvZjZwIW_Vzm-1uvf8z_PUfvQtHrV14dp", abi: "get_vault_data" })
+  let tonTVL = +vault[1] + +vault[8]
```

Comment:
hi @Cattontech thanks for the PR
I see that vault[1] is value of ton in the vault, what is vault[8] ?

Reply:
Hi @g1nt0ki  Thanks for your comment!
vault[8] is total transaction fee fee value in our protocol

Reply:
where does it get stored/ which contract holds it?

Reply:
It's calculator base on cTon (syth token), so we remove it from calculator tvl,
Thanks